tests/patmat/t4661.scala

```diff
@@ -0,0 +1,61 @@
+trait Foo
```

Comment:
@Blaisorblade I can break this up into smaller tests, if you want. Not sure want the convention is.